### PR TITLE
Update update-phone-number-and-email-address.md

### PR DESCRIPTION
**Pull request by msbemba**
Change in accessing my account

### CODE REVIEW

File: Office365-Admin/manage/update-phone-number-and-email-address.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Use the **Personal info** page to change your name, mobile phone number, and alt
   
 1. Browse to the Microsoft 365 admin center.
     
-2. Click **Settings** ![Settings](../media/4b83e9cb-c7e4-46c8-b3d1-cfee017123ae.png) \> **Office 365 Settings** \> **Personal info**.<br/>![Update your admin personal info](../media/de422327-dd57-47c5-a3a0-3ddbae805bb8.png)
+2. Click **Your Name** on the Navigation bar \> **My account** \> **Personal info**.<br/>![Update your admin personal info](../media/de422327-dd57-47c5-a3a0-3ddbae805bb8.png)
```
Comment (cmcatee-MSFT) [inline Office365-Admin/manage/update-phone-number-and-email-address.md]: 2. In the header, choose your profile icon \> **My account** \> **Personal info**.

File: Office365-Admin/manage/update-phone-number-and-email-address.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Use the **Personal info** page to change your name, mobile phone number, and alt
...
   
 3. On the **contact details** section, update your phone, primary email address and alternate email address. Make sure you use something different from your Office 365 email address for your alternate email address.<br/>![Use contact details to update admin info](../media/f3e52395-e259-426b-9f50-679bceeb987e.png)<br/>**IMPORTANT:** The alternate email address and the mobile phone number are needed for resetting your Office 365 admin password (not your computer admin password).
```
Comment (cmcatee-MSFT) [inline Office365-Admin/manage/update-phone-number-and-email-address.md]: 3. In the **contact details** section, choose **Edit**. Update your mobile, phone, and alternate email address. Make sure you use something different from your Office 365 email address for your alternate email address.
<br/>**IMPORTANT:** The alternate email address and the mobile phone number are needed for resetting your Office 365 admin password (not your computer admin password).

File: Office365-Admin/manage/update-phone-number-and-email-address.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ Use the **Personal info** page to change your name, mobile phone number, and alt
...
```
Comment (cmcatee-MSFT) [inline Office365-Admin/manage/update-phone-number-and-email-address.md]: 4. When you are finished, choose **Save**.

Comment (msbemba) [reply Office365-Admin/manage/update-phone-number-and-email-address.md]: @cmcatee-MSFT I have made the changes. Thank you.

**Comment by mypil** [comment]
@cmcatee-MSFT - Your recommended changes has been applied to this PR so it is now ready for your final review for merging. Thank you.

File: Office365-Admin/manage/update-phone-number-and-email-address.md [hunk 4]
```diff
@@ -31,9 +31,11 @@ Use the **Personal info** page to change your name, mobile phone number, and alt
   
 1. Browse to the Microsoft 365 admin center.
     
-2. Click **Settings** ![Settings](../media/4b83e9cb-c7e4-46c8-b3d1-cfee017123ae.png) \> **Office 365 Settings** \> **Personal info**.<br/>![Update your admin personal info](../media/de422327-dd57-47c5-a3a0-3ddbae805bb8.png)
+2. In the header, choose your profile icon \> **My account** \> **Personal info**.<br/>![Update your admin personal info](../media/de422327-dd57-47c5-a3a0-3ddbae805bb8.png)
```
Comment (cmcatee-MSFT) [inline Office365-Admin/manage/update-phone-number-and-email-address.md]: Remove the image

File: Office365-Admin/manage/update-phone-number-and-email-address.md [hunk 5]
```diff
@@ -31,9 +31,11 @@ Use the **Personal info** page to change your name, mobile phone number, and alt
...
   
-3. On the **contact details** section, update your phone, primary email address and alternate email address. Make sure you use something different from your Office 365 email address for your alternate email address.<br/>![Use contact details to update admin info](../media/f3e52395-e259-426b-9f50-679bceeb987e.png)<br/>**IMPORTANT:** The alternate email address and the mobile phone number are needed for resetting your Office 365 admin password (not your computer admin password). 
+3. In the **contact details** section, choose Edit. Update your mobile, phone, and alternate email address. Make sure you use something different from your Office 365 email address for your alternate email address.<br/>![Use contact details to update admin info](../media/f3e52395-e259-426b-9f50-679bceeb987e.png)<br/>**IMPORTANT:** The alternate email address and the mobile phone number are needed for resetting your Office 365 admin password (not your computer admin password).
```
Comment (cmcatee-MSFT) [inline Office365-Admin/manage/update-phone-number-and-email-address.md]: Remove the image

File: Office365-Admin/manage/update-phone-number-and-email-address.md (hunk 4 above)
Comment (msbemba) [reply Office365-Admin/manage/update-phone-number-and-email-address.md]: removed the image

File: Office365-Admin/manage/update-phone-number-and-email-address.md (hunk 5 above)
Comment (msbemba) [reply Office365-Admin/manage/update-phone-number-and-email-address.md]: Removed the image

**Comment by msbemba** [comment]
@cmcatee-MSFT I have removed the image as per your instructions.

**Pull request merged**